Extension/package.json

```diff
@@ -146,6 +146,12 @@
           "description": "Controls whether inactive preprocessor blocks are colored differently than active code. This setting is ignored by the Tag Parser engine.",
           "scope": "resource"
         },
+        "C_Cpp.inactiveRegionOpacity": {
+          "type:": "number",
+          "default": 0.5,
```

Comment:
Can you add `"minimum": 0` and `"maximum": 1` to the schema so that VS Code will squiggle values outside the valid range?

Reply:
Added!